static/js/src/app/campaign_results.js

```diff
@@ -357,14 +357,15 @@ var renderDevice = function (event_details) {
 }
 
 function renderTimeline(data) {
+    console.log(data)
     record = {
         "first_name": data[2],
         "last_name": data[3],
         "email": data[4],
         "position": data[5],
         "status": data[6],
-        "send_date": data[7],
-        "reported": data[8]
+        "reported": data[7],

```

Comment:
Whoops! This was a bug that's apparently been there since we introduced reporting. I went ahead and fixed it here.